src/artemis/tests/test_fast_grid_scan_plan.py

```diff
@@ -135,9 +137,13 @@ def test_results_passed_to_move_motors(bps_mv: MagicMock):
     )
 
 
-@patch("artemis.external_interaction.communicator_callbacks.wait_for_result")
-@patch("artemis.external_interaction.communicator_callbacks.run_end")
-@patch("artemis.external_interaction.communicator_callbacks.run_start")
+@patch(
+    "artemis.external_interaction.zocalo_interaction.ZocaloHandlerCallback.wait_for_result"
+)
+@patch("artemis.external_interaction.zocalo_interaction.ZocaloHandlerCallback.run_end")
+@patch(
+    "artemis.external_interaction.zocalo_interaction.ZocaloHandlerCallback.run_start"
+)
```

Comment:
Should: As above, does it make sense to mock all these out now? We can just mock out the `ZocaloHandlerCallback` as a `MagicMock`